src/serviceFacades/instantlaunches.js

```diff
@@ -436,6 +441,19 @@ export const instantlyLaunch = ({ instantLaunch, resource, output_dir }) => {
             submission.output_dir = output_dir;
             return submission;
         })
+        .then((submission) => {
+            const notify =
+                preferences?.enableAnalysisEmailNotification || false;
+            submission.notify = notify;
+            const notifyPeriodic =
+                !!preferences?.enablePeriodicEmailNotification;
+            submission.notify_periodic = notifyPeriodic;
+            const periodicPeriod =
+                preferences?.periodicNotificationPeriod || 14400;
+            submission.periodic_period = periodicPeriod;
+
+            return submission;
+        })
```

Comment:
This works fine as-is, but here's another more compact way to modify this submission JSON:
```suggestion
        .then((submission) => ({
            ...submission,
            notify: !!preferences?.enableAnalysisEmailNotification,
            notify_periodic: !!preferences?.enablePeriodicEmailNotification,
            periodic_period: preferences?.periodicNotificationPeriod || 14400,
        }))
```

Reply:
This is basically the code I was wondering if I should encapsulate somewhere else (since there is similar `const periodicPeriod = preferences?.periodicNotificationPeriod || 14400;` type code in the app launch code from the last PR) , but condensing it this way is better than doing that, I think.

Reply:
I see now. That's a good instinct, but this is probably fine for now. If we need to grab these preferences with these defaults in a 3rd place, then it will definitely be time to refactor 😃